src/styles/overrides/_ie-hacks.scss

```diff
@@ -26,3 +26,6 @@
     background: $pale-blue;
   }
 }
+.ms-focus-within {
```

Comment:
This needs to move into the media query, otherwise it will be available in all browsers.